graphene/types/mutation.py

```diff
@@ -50,7 +50,7 @@ def __init_subclass_with_meta__(cls, resolver=None, output=None, arguments=None,
                     warn_deprecation((
                         "Please use {name}.Arguments instead of {name}.Input."
                         "Input is now only used in ClientMutationID.\n"
-                        "Read more: https://github.com/graphql-python/graphene/blob/2.0/UPGRADE-v2.0.md#mutation-input"
+                        "Read more: https://github.com/graphql-python/graphene/blob/v2.0.0/UPGRADE-v2.0.md#mutation-input"
```

Comment:
This line is over the 120 character limit. Could you split it up?

Reply:
Sure!

Reply:
Sure thing!